apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SessionPoolImpl.cs

```diff
@@ -205,12 +207,22 @@ private void Push(SessionPoolEntry entry)
                     sw.Start();
                 }
                 //create a new session, blocking or throwing if at the limit.
-                var result = await Key.Client.CreateSessionAsync(new DatabaseName(Key.Project, Key.Instance, Key.Database), cancellationToken).ConfigureAwait(false);
-                if (sw != null)
+                await _sessionCreateThrottle.WaitAsync(cancellationToken).ConfigureAwait(false);
```

Comment:
and actually there is also a minor issue here in that the perf counter is including the throttle time -- which is 'ok', but I'd prefer to have the raw times to the server since its been known to have issues.